src/polyswarm_api/__init__.py

```diff
@@ -116,8 +58,12 @@ def __init__(self, key, uri="https://api.polyswarm.network/v1", get_limit=100,
 
         self.get_semaphore = asyncio.Semaphore(get_limit)
 
-        # ...sigh
-        self.engine_resolver = EngineResolver(self.network)
+        self.network = "prod" if uri.find("lb.kb") == -1 else "lb.kb"
+
+        # TODO does this need commmunity?
+        self.portal_uri = "https://polyswarm.network/scan/results/" if self.network == "prod" else "https://portal.stage.polyswarm.network/"
```

Comment:
Portal probably *should* be putting that in the URL, but for now it doesn't so this reflects that.